crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -444,7 +451,19 @@ define_tedge_config! {
                 #[tedge_config(note = "If set to 'auto', this cleans the local session accordingly the detected version of mosquitto.")]
                 #[tedge_config(example = "auto", default(variable = "AutoFlag::Auto"))]
                 local_cleansession: AutoFlag,
-            }
+            },
+
+            #[tedge_config(default(value = false))]
+            #[doku(skip)] // Hide the configuration in `tedge config list --doc`
+            in_mapper: bool,
```

Comment:
Why hide this setting?

Reply:
I want to get a bit further with the feature before finalising on what configurations we expose, especially while there are rough edges.